gq/benchmark_test.go

```diff
@@ -34,20 +35,17 @@ type testTuple struct {
 func BenchmarkSigning(b *testing.B) {
 	// Generate test matrix
 	matrix, err := generateTestMatrix(b.N)
-	if err != nil {
-		b.Fatal(err)
-	}
+	require.NoError(b, err)
 
 	// Reset the benchmark timer to exclude setup time
 	b.ResetTimer()
 
 	var signerVerifier SignerVerifier
 	for i := 0; i < b.N; i++ {
 		signerVerifier, err = NewSignerVerifier(matrix[i].rsaPublicKey, 256)
-		if err != nil {
-			b.Fatal(err)
-		}
+		require.NoError(b, err)
 		_, err = signerVerifier.SignJWT(matrix[i].token)
+		require.NoError(b, err)
```

Comment:
The error for this test was originally ignored, not sure if this was on purpose but I handled it for now. Please advise whether to keep it or remove it

Reply:
I think it is worth checking just in case and to reduce the number of warnings.